src/Features/Admin/Categories/epics.js

```diff
@@ -0,0 +1,25 @@
+import { ofType } from 'redux-observable';
+import { of } from 'rxjs';
+import { ajax } from 'rxjs/ajax';
+import { mergeMap, catchError } from 'rxjs/operators';
+
+import { FETCH_CATEGORIES } from './ActionTypes';
+
+export const fetchCategoriesEpic = (action$, state) =>
+  action$.pipe(
+    ofType(FETCH_CATEGORIES),
+    mergeMap(({ payload }) => {
+      return ajax({
+        url: 'http://localhost:4000/categories',
+        method: 'GET',
+      }).pipe(
+        mergeMap((res) => {
+          payload(res.response);
```

Comment:
Don't we need the data in store after API calls successfully runs ??  you did not return the action from mergeMap .